.github/actions/remote-release/chart_version_from_tag.py

```
@@ -0,0 +1,48 @@
+import re
+import sys
+
+if len(sys.argv) < 2 or sys.argv[1] == "":
+    raise Exception("Chart.yaml path should be passed as first argument")
+
+if len(sys.argv) < 3 or sys.argv[2] == "":
+    raise Exception("Release tag should be passed as second argument")
+
+chart_yaml_path = sys.argv[1]
+release_tag = sys.argv[2]
+
+# Extract version from tag (v0.1.2 -> 0.1.2)
+new_version = release_tag.lstrip("v")
+
+with open(chart_yaml_path, "r") as chart_file:
+    chart_yaml = chart_file.read()
+
+updated_yaml = chart_yaml
+
+# Update version to match release tag
+match = re.search(r"version:\s*(.+)", chart_yaml)
+if match:
```

What happens if version match is None? I see that it's a silent failure, should we throw an exception and fail properly?

---

Hmm, yeah it might be good to add a safe check, but AFAIK Chart.yaml without a version is invalid and will fail in packaging

---

i will make the regexp case insensitive in case someone will put `Version` in chart